src/overlays/actors/ovl_En_Ossan/z_en_ossan.c

```diff
@@ -20,227 +129,2379 @@ const ActorInit En_Ossan_InitVars = {
+                temp_v0 = D_80AC8C9C[sp24[i].funcIdx](sp24[i].funcIdx);
+                tana = &this->tana;
+                if (temp_v0 >= 0) {
+                    this->unk_200[i] = (EnGirlA*)Actor_Spawn(
+                        &globalCtx->actorCtx, globalCtx, ACTOR_EN_GIRLA, sp24[i].pos.x + (*tana)->actor.posRot.pos.x,
+                        sp24[i].pos.y + (*tana)->actor.posRot.pos.y, sp24[i].pos.z + (*tana)->actor.posRot.pos.z,
+                        (*tana)->actor.shape.rot.x, (*tana)->actor.shape.rot.y + D_80AC88F4[i],
+                        (*tana)->actor.shape.rot.z, temp_v0);
+                }
+            }
+        }
+    }
+}
+
+void func_80AC3098(GlobalContext* globalCtx) {
+    func_8010B720(globalCtx, 0x9E);
+}
+
+void func_80AC30B8(GlobalContext* globalCtx) {
+    if (globalCtx->curSpawn == 0) {
+        func_8010B720(globalCtx, 0x5046);
+    } else {
+        func_8010B720(globalCtx, 0x504E);
+    }
+}
+
+void func_80AC30FC(GlobalContext* globalCtx) {
+    func_8010B720(globalCtx, 0x504E);
+}
+
+void func_80AC311C(GlobalContext* globalCtx) {
+    func_8010B720(globalCtx, 0x10BA);
+}
+
+void func_80AC313C(GlobalContext* globalCtx) {
+    if (globalCtx->curSpawn == 0) {
+        func_8010B720(globalCtx, 0x9D);
+    } else {
+        func_8010B720(globalCtx, 0x9C);
+    }
+}
+
+void func_80AC3180(GlobalContext* globalCtx) {
+    func_8010B720(globalCtx, 0x7076);
+}
+
+void func_80AC31A0(GlobalContext* globalCtx) {
+    if (LINK_AGE_IN_YEARS == YEARS_CHILD) {
+        func_8010B720(globalCtx, 0x403A);
+    } else {
+        func_8010B720(globalCtx, 0x403B);
+    }
+}
+
+void func_80AC31F4(GlobalContext* globalCtx) {
+    if (LINK_AGE_IN_YEARS == YEARS_CHILD) {
+        if (gSaveContext.eventChkInf[2] & 0x20) {
+            func_8010B720(globalCtx, 0x3028);
+        } else if (CUR_UPG_VALUE(UPG_STRENGTH) != 0) {
+            func_8010B720(globalCtx, 0x302D);
+        } else {
+            func_8010B720(globalCtx, 0x300F);
+        }
+    } else if (!CHECK_QUEST_ITEM(QUEST_MEDALLION_FIRE)) {
+        func_8010B720(globalCtx, 0x3057);
+    } else {
+        func_8010B720(globalCtx, 0x305B);
+    }
+}
+
+void func_80AC32C4(GlobalContext* globalCtx) {
+    if ((gSaveContext.itemGetInf[3] & 0x100) && (gSaveContext.itemGetInf[3] & 0x200) &&
+        (gSaveContext.itemGetInf[3] & 0x400) && (gSaveContext.itemGetInf[3] & 0x800)) {
+        func_8010B720(globalCtx, 0x70AE);
+    } else if (globalCtx->msgCtx.choiceIndex != 0) {
+        if (globalCtx->msgCtx.choiceIndex == 1) {
+            func_8010B720(globalCtx, 0x70A4);
+        }
+    } else {
+        func_8010B720(globalCtx, 0x70A3);
+    }
+}
+
+void func_80AC3350(EnOssan* this, GlobalContext* globalCtx, f32 arg2) {
+    this->unk_2D4 = arg2;
+    Camera_SetCameraData(ACTIVE_CAM, 0xC, NULL, NULL, (s32)arg2, 0, 0);
+}
+
+s32 func_80AC33B0(EnOssan* this, GlobalContext* globalCtx, s16* arg2) {
+    if (arg2[1] != 0x192) {
```

Comment:
this is an object so you can use the enum

Reply:
0x192 doesn't exist in z64object.h. Ends at 0x191.